core/src/test/resources/test_example.ldif

```diff
@@ -96,20 +96,26 @@ dn: cn=group1,ou=Groups,dc=example,dc=com
 objectClass: top
 objectClass: groupOfUniqueNames
 cn: group1
+o: Group 1
```

Comment:
why `o`? maybe we could use a more descriptive key?

Reply:
We already use `ou` and I didn't want to confuse this attribute. As you can see, `ou` `o`, and `description` are valid attributes, and `o` seemed best for our case. Do you have any other option?

![image](https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/assets/26696870/07090cb1-4bd4-47d1-a5b1-9634d6e5f80e)
![image](https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/assets/26696870/9ba49666-0118-481c-a933-1aa881de4159)
![image](https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/assets/26696870/a76e5a16-7637-4de4-a319-0dd5d46d6679)

Reply:
oh, OK. Didn't know that. So, let's leave it like this :)